.travis.yml

```diff
@@ -3,6 +3,10 @@ node_js:
   - "4"
   - "5"
 
+script:
+  - npm run lint
+  - npm run style
```

Comment:
that's great but you're gonna now need to also add `npm test` as you've overwritten the default command